src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -36,6 +36,8 @@ var HTMLDOMPropertyConfig = {
     allowFullScreen: HAS_BOOLEAN_VALUE,
     allowTransparency: 0,
     alt: 0,
+    /* 'as' is introduced to links with 'preload' type: https://w3c.github.io/preload/ */
```

Comment:
nitpick, but can you use a single line comment instead of a block comment? Just so it's consistent. Also, I think you can just remove the link to the W3C standard 👍